spec/features/idv/steps/in_person/verify_step_spec.rb

```diff
@@ -186,4 +186,123 @@
+      expect(page).to have_text(InPersonHelper::GOOD_IDENTITY_DOC_ADDRESS2).twice
+      expect(page).to have_text(InPersonHelper::GOOD_IDENTITY_DOC_CITY).twice
+      expect(page).to have_text(
+        Idp::Constants::
+                MOCK_IDV_APPLICANT_STATE_ID_ADDRESS[:identity_doc_address_state],
+      ).twice
+
+      expect(page).to have_text(InPersonHelper::GOOD_IDENTITY_DOC_ZIPCODE).twice
+      expect(page).to have_text(DocAuthHelper::GOOD_SSN_MASKED)
+    end
+  end
+
+  context 'with in person verify info controller not enabled' do
+    let(:capture_secondary_id_enabled) { true }
+    let(:enrollment) { InPersonEnrollment.new(capture_secondary_id_enabled:) }
+    let(:user) { user_with_2fa }
+    let(:same_address_as_id) { true }
+    let(:double_address_verification) { true }
+
+    before do
+      allow(IdentityConfig.store).to receive(:in_person_capture_secondary_id_enabled).
+        and_return(true)
+      allow(IdentityConfig.store).to receive(:in_person_verify_info_controller_enabled).
+        and_return(false)
+    end
+
+    it 'displays expected headers & data on /verify',
+       allow_browser_log: true do
+      sign_in_and_2fa_user(user)
+      begin_in_person_proofing(user)
+      complete_prepare_step(user)
+      complete_location_step(user)
+      complete_state_id_step(
+        user, same_address_as_id: same_address_as_id,
+              double_address_verification: double_address_verification
+      )
+      click_idv_continue
+      complete_ssn_step(user)
+
+      # confirm url is /verify
+      expect(page).to have_current_path(idv_in_person_step_url(step: :verify))
+
+      # verify page
+      expect(page).to have_content(t('headings.verify'))
+
+      # confirm headers are on template
+      expect(page).to have_content(t('headings.state_id').tr(' ', ' '))
+      expect(page).to have_content(t('headings.residential_address'))
+      expect(page).to have_content(t('headings.ssn'))
+
+      # confirm data is on template
+      expect(page).to have_text(InPersonHelper::GOOD_FIRST_NAME)
+      expect(page).to have_text(InPersonHelper::GOOD_LAST_NAME)
+      expect(page).to have_text(InPersonHelper::GOOD_DOB_FORMATTED_EVENT)
+      expect(page).to have_text(
+        "#{I18n.t('idv.form.issuing_state')}: #{Idp::Constants::
+        MOCK_IDV_APPLICANT_STATE_ID_JURISDICTION}",
+      )
+      expect(page).to have_text(InPersonHelper::GOOD_STATE_ID_NUMBER)
+      expect(page).to have_text(InPersonHelper::GOOD_IDENTITY_DOC_ADDRESS1).twice
+      expect(page).to have_text(InPersonHelper::GOOD_IDENTITY_DOC_ADDRESS2).twice
+      expect(page).to have_text(InPersonHelper::GOOD_IDENTITY_DOC_CITY).twice
+      expect(page).to have_text(
+        Idp::Constants::
+                MOCK_IDV_APPLICANT_STATE_ID_ADDRESS[:identity_doc_address_state],
+      ).twice
+      expect(page).to have_text(InPersonHelper::GOOD_IDENTITY_DOC_ZIPCODE).twice
+      expect(page).to have_text(DocAuthHelper::GOOD_SSN_MASKED)
+    end
+  end
```

Comment:
I did not test capture_secondary_id_enabled true vs false. I feel like this has already been tested. I feel like line 204 shows when feature flags are enabled- we have additional headers and data on the page- and that we are on /verify_info